internal/identify/studio_test.go

```diff
@@ -87,18 +105,6 @@ func Test_createMissingStudio(t *testing.T) {
 			&createdID,
 			false,
 		},
-		{
-			"invalid stash id",
-			args{
-				invalidEndpoint,
-				&models.ScrapedStudio{
-					Name:         validName,
-					RemoteSiteID: &remoteSiteID,
-				},
-			},
-			nil,
-			true,
-		},
```

Comment:
With the removal of the UpdateStashIDs method, this test isn't really applicable.